### PR TITLE
feat: 🎸 Add ToolbarRefresher component to sessions in admin

### PR DESCRIPTION
**Pull request by cameronperera**
✅ Closes: ICU-4133

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-4133)

## Description

:technologist: [Admin preview](https://boundary-ui-git-icu-4133-show-timestamp-in-ses-a69eb8-hashicorp.vercel.app/scopes/s_xk1i4dk50v2/sessions)

:desktop_computer: [Desktop preview](https://boundary-ui-desktop-git-icu-4133-show-timestam-b305aa-hashicorp.vercel.app/scopes/global/projects/sessions)

### Screenshots (if appropriate):
toolbar-refresher-admin

### CODE REVIEW

File: ui/admin/app/templates/scopes/scope/sessions/index.hbs [hunk 1]
```diff
@@ -61,9 +61,9 @@
               >{{t 'actions.clear-all-filters'}}
               </Rose::Button>
             {{/if}}
-            <LoadingButton @onClick={{route-action 'refreshSessions'}}>
-              {{t 'actions.refresh'}}
-            </LoadingButton>
+
```
Comment (ZedLi) [inline ui/admin/app/templates/scopes/scope/sessions/index.hbs:64]: Should there be whitespace here? Does `prettier` check this?

**Review by ZedLi — APPROVED** [review]
LGTM!

File: ui/admin/app/templates/scopes/scope/sessions/index.hbs (hunk 1 above)
Comment (cameronperera) [reply ui/admin/app/templates/scopes/scope/sessions/index.hbs:64]: Prettier did not seem to mind for me. I was following suit with what was already in place when it was added to Sessions in desktop. As well I think it helps bring attention to a single line of code that is easily missed without the whitespace. However, I am open to removing it.

**Review by calcaide — APPROVED** [review]
LGTM!

**Pull request merged**